Resources/Maps/_Mono/Shuttles/framework.yml

```
@@ -0,0 +1,564 @@
+meta:
+  format: 7
+  category: Grid
+  engineVersion: 247.2.0
+  forkId: ""
+  forkVersion: ""
+  time: 03/30/2025 07:37:30
+  entityCount: 78
+maps: []
+grids:
+- 2
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 0

(indentation)